tests/main/auto-refresh-retry/task.yaml

```diff
@@ -41,15 +42,15 @@ execute: |
 
     echo "wait for auto-refresh to happen and fail"
     for _ in $(seq 120); do
-        if get_journalctl_log | MATCH "state ensure error"; then
+        if get_journalctl_log | MATCH "state ensure error: persistent network error"; then
             break
         fi
         echo "Ensure refresh"
         snap debug ensure-state-soon
         sleep 5
     done
 
-    journalctl | MATCH "state ensure error: persistent network error"
+    get_journalctl_log | MATCH "state ensure error: persistent network error"
```

Comment:
I remember having issues with this test and get_journal_log (would fail sometimes on some systems even though there was a matching line in the logs - perhaps something with log cursors).

Reply:
@stolowski, thanks for the review, in this case we shouldnt make journalctl becuase it brings the whole history, and that string could match with something that happend on another test which ran before.